prowler/providers/aws/services/s3/s3_service.py

```diff
@@ -262,9 +262,12 @@ def __get_bucket_tagging__(self, bucket):
                 "TagSet"
             ]
             bucket.tags = bucket_tags
-        except ClientError as e:
-            if e.response["Error"]["Code"] == "NoSuchTagSet":
-                bucket.tags = []
+        except ClientError as error:
+            bucket_tags = []
```

Comment:
I think you need to assign this to the `bucket` object right?

Reply:
Good catch!!!! Thank you